### DISCUSSION TITLE
Welcome to the WooCommerce core repository discussions!

### DISCUSSION THREAD

**Discussion by peterfabian** [woocommerce/woocommerce#32015] [Announcement]
# 👋 Welcome to the discussions area of the WooCommerce monorepository!

To make sure we work on the best version of WooCommerce we can for people who use it, we're going to use Requests for Comments more often. We'd love all of you to join the discussions about the features we propose, or the way you would like to see the WooCommerce platform evolve. 

That is why we're starting an experiment with enabling the **Requests for Comments** here in GitHub instead of using the [developer portal's blog](https://developer.woocommerce.com/blog/), as it's closer to the developer community we're addressing, allows for structured discussion and direct code referencing, and hopefully makes it easier to follow the discussion with upvotes for comments. At least initially, we will announce the RFCs also on the [developer portal's blog](https://developer.woocommerce.com/blog/) to make sure people are aware of this new feature.

We hope that you find this useful and productive and welcome you to voice your opinions on the RFCs! 🙌

**Comment by konnectikaBNC** [comment]
Hi,
I bought a plugin from Woo Commerce, Rental Products and 2 have a glitches.
1. For the option Rental or Purchase -> When i have a stock = 1 for a product, client rents it for a number of days, pays for the rent. After this, if client goes to purchase the product (which was stock =1 before being rented, then stock = 0 after it was rented) he still has the option to buy it which is not logical. What i need is the purchase button to be disabled when a product is rented and the stock become 0. I do not want the product to disappear from the website, i need it to remain there so the client can see it 
2. After a successfully of 2 products rental payment i decided to cancel that order because i found out some issues with those products and refund the money to the client. When i checked the shop, the products has the availability in red meaning that the message was that they were not available for the client but when i click on Rent button and try to pock the dates, those dates were available for choosing
![chrome_Fy3mn273X9](https://user-images.githubusercontent.com/57504852/182981481-fe88f6c6-950c-4c36-b713-fc7aeb144bc5.png)



.

**Reply by peterfabian** [reply]
Hi, please contact our support where they can help you troubleshoot the issue: https://woocommerce.com/my-account/create-a-ticket/

**Comment by niagaku909** [comment]
![Gambar WhatsApp 2024-10-23 pukul 09 31 36_9079f037](https://github.com/user-attachments/assets/206ae05e-f6ec-4a66-b5ec-97c1f4f885b6)

i have checkout page problem,how i can fix it
